warning: `QDirModel::QDirModel()` is deprecated: Use QFileSystemModel

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/InputFile.cpp: In constructor ‘OpenMS::InputFile::InputFile(QWidget*)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/InputFile.cpp:57:48: warning: ‘QDirModel::QDirModel(QObject*)’ is deprecated: Use QFileSystemModel [-Wdeprecated-declarations]
   57 |     completer->setModel(new QDirModel(completer));
      |                                                ^
In file included from /usr/include/qt/QtWidgets/QDirModel:1,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/InputFile.cpp:44:
/usr/include/qt/QtWidgets/qdirmodel.h:73:67: note: declared here
   73 |     QT_DEPRECATED_VERSION_X_5_15("Use QFileSystemModel") explicit QDirModel(QObject *parent = nullptr);
      |                                                                   ^~~~~~~~~
```

--- a/src/openms_gui/source/VISUAL/InputFile.cpp
+++ b/src/openms_gui/source/VISUAL/InputFile.cpp
@@ -41,7 +41,7 @@
 #include <QtWidgets/QMessageBox>
 #include <QtWidgets/QFileDialog>
 #include <QtWidgets/QCompleter>
-#include <QtWidgets/QDirModel>
+#include <QFileSystemModel>
 #include <QDragEnterEvent>
 #include <QMimeData>
 
@@ -54,7 +54,7 @@ namespace OpenMS
   {
     ui_->setupUi(this);
     QCompleter* completer = new QCompleter(this);
-    completer->setModel(new QDirModel(completer));
+    completer->setModel(new QFileSystemModel(completer));
     ui_->line_edit->setCompleter(completer);
     connect(ui_->browse_button, SIGNAL(clicked()), this, SLOT(showFileDialog()));
   }

--- a/src/openms_gui/source/VISUAL/OutputDirectory.cpp
+++ b/src/openms_gui/source/VISUAL/OutputDirectory.cpp
@@ -42,7 +42,7 @@
 #include <QtWidgets/QMessageBox>
 #include <QtWidgets/QFileDialog>
 #include <QtWidgets/QCompleter>
-#include <QtWidgets/QDirModel>
+#include <QFileSystemModel>
 
 
 namespace OpenMS
@@ -53,7 +53,7 @@ namespace OpenMS
   {
     ui_->setupUi(this);
     QCompleter* completer = new QCompleter(this);
-    QDirModel* dir_model = new QDirModel(completer);
+    QFileSystemModel* dir_model = new QFileSystemModel(completer);
     dir_model->setFilter(QDir::AllDirs);
     completer->setModel(dir_model);
     ui_->line_edit->setCompleter(completer);

